dotorg/dotorg-support-capacity-report-collector.user.js

```diff
@@ -116,4 +132,6 @@
 * Fixes: Update and Download URL
 * [1.0.1] 2024-04-23
 * Updates: Last Replies + Author
+* [1.0.2] 2024-07-08
```

Comment:
I personally like the changelog to be in descending order and with a bit more spacing like we do in the plugin readme.txt files. I will leave it up to you.